packages/client/ui/react-ui/src/providers/CrossmintWalletProvider.tsx

```diff
@@ -151,6 +197,22 @@ export function CrossmintWalletProvider({
                           document.body
                       )
                     : null}
+                {phoneSignerDialogOpen && phoneNumber
```

Comment:
phoneNumber != null

Reply:
switched to experimental_customAuth?.phone != null